<launch>
  <node pkg="project1" type="talkerGoals" name="talkerGoals" output="screen">
    <param name="X_GOAL" value="8.0"/>
    <param name="Y_GOAL" value="6.0"/>
    <param name="T_GOAL" value="5.0"/>
  </node>

  <!-- Generate world with goal area -->
  <node pkg="project1" type="world_generator" name="world_generator" output="screen"/>

  <!-- Launch Stage with the generated world -->
  <node name="Stage" pkg="stage_ros" type="stageros" args="$(find project1)/worlds/mundo.world"/>

  <node pkg="project1" type="moveRobot" name="moveRobot0" output="screen">
    <param name="ID_ROBOT" value="0"/>
	<param name="LEADER" value="-1"/>
    <param name="ALGOR" value="2"/>
  </node>

  <node pkg="project1" type="moveRobot" name="moveRobot1" output="screen">
    <param name="ID_ROBOT" value="1"/>
	<param name="LEADER" value="0"/>
    <param name="ALGOR" value="2"/>
  </node>

  <node pkg="project1" type="moveRobot" name="moveRobot2" output="screen">
    <param name="ID_ROBOT" value="2"/>
	<param name="LEADER" value="1"/>
    <param name="ALGOR" value="1"/>
  </node>

  <node pkg="project1" type="moveRobot" name="moveRobot3" output="screen">
    <param name="ID_ROBOT" value="3"/>
	<param name="LEADER" value="0"/>
    <param name="ALGOR" value="1"/>
  </node>
</launch>
